app/viewmodels/TradeJourneyStatusesViewModel.scala

```diff
@@ -43,9 +44,13 @@ object TradeJourneyStatusesViewModel {
       if (optJourney.isEmpty) None else optJourney.head.completedState
     }
 
-    val (abroadCompletionStatus, incomeCompletionStatus) = (getStatus(Abroad), getStatus(Income))
+    val (abroadCompletionStatus, incomeCompletionStatus, expensesCompletionStatus) = (
+      getStatus(Abroad),
```

Comment:
getStatus return Option[Boolean] is hard to undersatnd by reading the method name and checking the return type what it express.

can we return proper Status like Completed or InProgress here?